include/linux/find_closest.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Find closest element functions
+ *
+ * Copyright (C) 2025 Alexandru Soponar <asoponar@taladin.ro>
+ * Based on util_macros.h implementation by Alexandru Ardelean <aardelean@baylibre.com>
+ */
+#ifndef _LINUX_FIND_CLOSEST_H_
+#define _LINUX_FIND_CLOSEST_H_
+
+#include <linux/types.h>
+
+/**
```

Comment:
i think doc-strings need to be in one place; not both;

since they are in the `find_closes.c`, i think these could be removed;